troposphere/static/js/public_site/AppRoutes.react.js

```diff
@@ -14,14 +15,14 @@ define(function (require) {
       ImagesMaster = require('components/images/ImagesMaster.react');
 
   var AppRoutes = (
-    <Route name="root" path="/image" handler={Master}>
+    <Route name="root" path="/application" handler={Master}>
       <Route name="images" handler={ImagesMaster}>
         <DefaultRoute name="search" handler={ImageListPage}/>
         <Route name="tags" handler={ImageTagsPage}/>
         <Route name="image-details" path=":imageId" handler={ImageDetailsPage}/>
       </Route>
       <Route name="help" handler={HelpPage}/>
-      <Redirect from="/image" to="/application/images"/>
+      <Redirect from="/application" to="/application/images"/>
     </Route>
```

Comment:
This is probably the detail that we need to get place on a "login" page... how to see the Image Catalog.